.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,205 @@
+    * When you are done making changes, review locally your changes with `git diff` or any git client, to make sure you are doing to the model what you intended.
+
+7. Commit your changes and push your branch to GitHub.
+    ```
+    git add .
+    git commit -m "Title of your commit"
+    git push origin name-of-your-branch
+    ```
+    [See below](#semantic-commits) for recommendations on how to name your commits. In case of larger updates, you can of course make several commits on a single contribution. However, if you need to do too many commits, consider if your contribution could be instead split into separate branches (making it easier for reviewing later).
+
+8. Submit a pull request through the GitHub website (https://help.github.com/articles/creating-a-pull-request-from-a-fork/) to the `devel` branch of the original SysBioChalmers repo (not your fork). We recommend ticking the box "Allow edits from maintainers" if you wish for us to be able to contribute directly to your branch (speeding-up the reviewing process).
+
+Note that steps 3, 4, 5 and 7 can be done, if you prefer, with any git client, such as [Github Desktop](https://desktop.github.com/).
+
+Finally, and for larger features that you want to work on collaboratively, you may consider to first request to join our development team to get write access to the repository so that you can create a branch directly in the main repository (or simply ask the administrator to create a branch for you). Once you have a new branch, you can push your changes directly to the main repository and when finished, submit a pull request from that branch to ``devel``. [See below](#development-team-guidelines) for more details.
+
+Thank you very much for contributing to yeast-GEM!
+
+#### Branching model
+
+* `devel`: Is the branch all pull-requests should be based on.
+
+* `master`: Is only touched by the administrator and is the branch with the tested & reviewed model that is released or ready for the next release.
+
+* `gh-pages`: Is only touched by the administrator and for maintaining the [landing page](http://sysbiochalmers.github.io/yeast-GEM/) of yeast-GEM.
+
+* `{chore, doc, feat, fix, refactor, style}/descriptive-name`: Any other branch created in the model. If you work on a fix, start the branch name with `fix/`, if you work on a feature, start the branch name with `feat/`. Examples: `fix/format_reactions` or `feat/new_algorithms`. [See below](#semantic-commits) for more details on the possible actions you can use.
+	
+#### Semantic commits
+
+Please use concise descriptive commit messages. Ideally, use semantic commit messages to make it easier to show what you are aiming to do:
+
+`action-object: brief description`
+* `action` = {`feat`, `fix`, `refactor`, `style`, `doc`, `chore`, `test`}
+* `object` = {`rxn`, `rxn.annot`, `rxn.prop`, `met`, `met.annot`, `met.prop`, `gene`, `gene.annot`, `comp`, `comp.annot`, `data`}
```

Comment:
I don't see that much value in the `object` part, you'd probably want to read the full commit message anyway, and to me this seems to make things more complex. But okay to keep it like this.

Reply:
Below it is actually mentioned that `object` is optional, maybe mention that here already as well.

Reply:
done